vitals/Main.java

```diff
@@ -1,23 +1,36 @@
 package vitals;
 
 public class Main {
-    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
-        if(temperature < 0 || temperature > 45) {
-            System.out.println("Temperature is out of range!");
-            return false;
-        } else if(soc < 20 || soc > 80) {
-            System.out.println("State of Charge is out of range!");
-            return false;
-        } else if(chargeRate > 0.8) {
-            System.out.println("Charge Rate is out of range!");
-            return false;
-        }
-        return true;
-    }
 
-    public static void main(String[] args) {
-        assert(batteryIsOk(25, 70, 0.7f) == true);
-        assert(batteryIsOk(50, 85, 0.0f) == false);
-        System.out.println("Some more tests needed");
+    public static void main(final String[] args) {
+        VitalBatteryTest vitalBatteryTest = new VitalBatteryTest();
+        Vitals vitals;
+
+        vitals = new Temperature(32f);
```

Comment:
Atleast 5 values of temperature needs to be checked - min and max threshold and values < min, >max and value in between threshold value.